src/Components/SmartComponents/CVEDetailsPage/CVEDetailsPage.js

```diff
@@ -46,11 +47,16 @@ const CVEDetailsPage = () => {
     const details = useSelector(
         ({ CVEDetailsPageStore }) => CVEDetailsPageStore.cveDetails
     );
-    const cveDetails = useMemo(() => createCveDetailsPage(details), [details]);
+    const clusterDetails = useSelector(
+        ({ CVEDetailsPageStore }) => CVEDetailsPageStore.clusterDetails
+    );
+
+    const cveDetails = useMemo(() => createCveDetailsPage({ ...details, clusterDetails }), [details]);
```

Comment:
If you open any CVE thas has at least 1 cluster affected, you won't see the message with the first render. If you navigate back and then forth to the CVE details page, the message will appear. This happens because `cveDetails` is not updated if `clusterDetails` changes.

```suggestion
    const cveDetails = useMemo(() => createCveDetailsPage({ ...details, clusterDetails }), [details, clusterDetails]);
```